ui/component/copyableStreamkeyUnmask/view.jsx

```diff
@@ -0,0 +1,41 @@
+// @flow
+import Button from 'component/button';
+import React from 'react';
+
+type Props = {
+  snackMessage: ?string,
+  doToast: ({ message: string }) => void,
+  label?: string,
+  primaryButton?: boolean,
+  name?: string,
+};
+
+export default function CopyableText(props: Props) {
+  const { doToast, primaryButton = false } = props;
+
+  function showStreamkey() {
+    if (document.getElementById('livestream-key').type === 'password') {
+      doToast({ message: __('Showing Stream Key') });
+      document.getElementById('livestream-key').setAttribute('type', 'text');
+      document.getElementById('streamkey-unmask').style.color = 'white';
+      document.getElementById('streamkey-unmask').innerText = 'Hide Stream key';
+    } else if (document.getElementById('livestream-key').type === 'text') {
+      doToast({ message: __('Hiding Stream Key') });
```

Comment:
It is not necessary to show a toast for when hiding and showing the key.

Reply:
Removed this Component entirely.